src/h5web/visualizations/shared/AxisSystem.tsx

```diff
@@ -19,30 +19,30 @@ function AxisSystem(props: Props): JSX.Element {
   const { camera, size } = useThree();
   const { width, height } = size;
 
-  const { axisDomains, showGrid } = useContext(AxisSystemContext);
+  const { abscissaConfig, ordinateConfig } = useContext(AxisSystemContext);
+  const abscissa = useAbscissaScale();
+  const ordinate = useOrdinateScale();
 
   // axisDomains are the complete domains. visibleDomains change with the camera
-  const [visibleDomains, setVisibleDomains] = useState<AxisDomains>(
-    axisDomains
-  );
-
-  // Finds the visible domains from the camera FOV (zoom and position).
-  const { abscissaScale, abscissaScaleFn } = useAbscissaScale();
-  const { ordinateScale, ordinateScaleFn } = useOrdinateScale();
+  const [visibleDomains, setVisibleDomains] = useState<[Domain, Domain]>([
```

Comment:
I simplified the type of the state here since `AxisDomains` was no longer used anywhere else.